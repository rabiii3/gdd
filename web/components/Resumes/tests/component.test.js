import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from '../../../store/configureStore.test';
import { Component } from '..';

describe('app |Â components | ResumesList | component', () => {
  describe('render', () => {
    const state = {
      people: {
        data: {
          '1': {
            _id: '1',
            firstname: 'james',
            lastname: 'bond',
            email: 'james.bond@gmail.com',
            status: 'desable',
            type: 'user',
            phoneNumber: '07737383939',
            color: 'red',
            roles: ['user'],
          },
        },
      },
      resumes: {
        data: [
          {
            _id: 2,
            status: 'pending',
            //updatedAt: new Date(),
            candidateId: 1,
            createdBy: '4',
            //createdAt: new Date(),
          },
        ],
      },
    };
    const props = {
      classes: {},
      resumes: [],
      people: {},
      actions: {},
      allSkills: {},
      location: {},
    };
    it('should be defined', () => {
      const store = configureStore(state, null);

      const wrapper = shallow(
        <Provider store={store}>
          <Component {...props} />
        </Provider>,
      );

      expect(wrapper).toBeDefined();
    });

    it('should match snapshot', () => {
      const store = configureStore(state, null);
      const wrapper = shallow(
        <Provider store={store}>
          <Component {...props} />
        </Provider>,
      );
      expect(wrapper).toMatchSnapshot();
    });
  });
});
