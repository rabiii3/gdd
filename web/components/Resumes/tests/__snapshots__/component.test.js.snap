// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app |Â components | ResumesList | component render should match snapshot 1`] = `
<Component
  actions={Object {}}
  allSkills={Object {}}
  classes={Object {}}
  location={Object {}}
  people={Object {}}
  resumes={Array []}
/>
`;
